Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -239,8 +249,8 @@ void isotropic_remeshing(const FaceRange& faces
     }
     remesher.equalize_valences();
     remesher.tangential_relaxation(smoothing_1d, nb_laplacian);
-    remesher.project_to_surface();
 
```

Comment:
can you please keep `project_to_surface()` on the same line to keep code clean? :-)